components/python/urllib3/urllib3-PYVER.p5m

```diff
@@ -0,0 +1,106 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2015, 2019, Oracle and/or its affiliates. All rights reserved.
+#
+
+set name=pkg.fmri \
+    value=pkg:/library/python/urllib3-$(PYV)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary \
+    value="HTTP library with thread-safe connection pooling, file post, and more."
+set name=pkg.description \
+    value="urllib3 is a powerful, sanity-friendly HTTP client for Python. It brings many critical features that are missing from the Python standard libraries including: 1) Thread safety; 2) Connection pooling; 3) Client-side SSL/TLS verification; 4) File uploads with multipart encoding; 5) Helpers for retrying requests and dealing with HTTP redirects; 6) Support for gzip and deflate encoding; 7) Proxy support for HTTP and SOCKS."
+set name=com.oracle.info.description value="the Python urllib3 module"
+set name=info.classification \
+    value=org.opensolaris.category.2008:Development/Python
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream value="Andrey Petrov <andrey.petrov@shazow.net>"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.arc-caseid value=PSARC/2015/475
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+#
```

Comment:
Should 2.7 version also depend on ipaddress? 
Something like 
`$(PYTHON_2.7_ONLY)depend type=require fmri=library/python/ipaddress-$(PYV)`

Reply:
No idea at all. I am complete python moron.

Reply:
look at requires file, I suppose it depends on it